### ISSUE TITLE
A11y: Custom Products - Step by step mode can select invisible fields

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5057]
Using a screenreader it is possible to select fields that should not yet be active and available in step-by-step mode. Elements can be selected and will be read out by screenreader, but cannot be interacted with.

!Screen Recording 2024-10-09 at 18.01.23.mov|width=1538,height=1306,alt="Screen Recording 2024-10-09 at 18.01.23.mov"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38815**



[created from NEXT-38815, comment 502096]
